src/sources/host_metrics/cgroups.rs

```diff
@@ -25,13 +25,22 @@ pub(crate) struct CGroupsConfig {
     /// The number of levels of the cgroups hierarchy for which to report metrics.
     ///
     /// A value of `1` means just the root or named cgroup.
-    #[derivative(Default(value = "100"))]
+    #[derivative(Default(value = "default_levels()"))]
+    #[serde(default = "default_levels")]
+    #[configurable(metadata(docs::examples = 1))]
+    #[configurable(metadata(docs::examples = 3))]
     levels: usize,
 
     /// The base cgroup name to provide metrics for.
+    #[configurable(metadata(docs::examples = "/"))]
+    #[configurable(metadata(docs::examples = "system.slice/snapd.service"))]
```

Comment:
`snapd`? 🤢

Reply:
Happy to swap this out if you have a better example :)

Reply:
I'm sure some poor soul is using snap and need to collect logs 😆